etc/release.sh

```diff
@@ -5,9 +5,6 @@
 # exit if any command fails
 set -e
 
-# ensure we are on main before releasing
```

Comment:
I updated the other release scripts a while back to remove this since we might need to release from branches sometimes rather than main